website/static/js/home-page/meetingsAndConferencesPlugin.js

```diff
@@ -22,7 +22,7 @@ var MeetingsAndConferences = {
                         ]
                     ),
                     m('.col-md-4.text-center',
-                        m('div',  m('a.btn.btn-info.btn-lg.m-v-xl', { style : 'box-shadow: 0 0 9px -4px #000;', type:'button',  href:'/meetings/', onclick: function() {
+                        m('div',  m('a.btn.btn-info.btn-lg.btn-view-meetings.m-v-xl.f-w-xl', { style : 'box-shadow: 0 0 9px -4px #000;', type:'button',  href:'/meetings/', onclick: function() {
```

Comment:
I think `btn-info` can be removed, now that you've added a class which overrides the colors.

I can make this change.